templates/terraform/environments/stage/user_access.tf

```diff
@@ -173,6 +173,32 @@ data "aws_iam_policy_document" "deployer_access" {
 
 
 locals {
+  # define Kubernetes policy for developer env deployment
+  k8s_developer_env_access = [
+    # to support developer environment
+    {
+      verbs      = ["create", "exec", "list", "get", "delete", "patch", "update", "watch"]
+      api_groups = ["*"]
+      resources = ["namespaces", "deployments", "deployments/scale", "configmaps", "pods", "pods/log", "pods/status", "pods/portforward", "pods/exec",
+        "jobs", "cronjobs",
+        "ingresses", "services", "serviceaccounts",
+        "replicasets", "horizontalpodautoscalers", "horizontalpodautoscalers/status"
+      ]
+    },
+    {
+      verbs      = ["get", "list", "watch"]
+      api_groups = ["*"]
+      resources = [ "daemonsets", "endpoints", "events",
```

Comment:
Why just get,list,watch on daemonsets and pdbs? Seems like they should have the verbs listed above.

Reply:
will provide more permissions to developers considering that Zero is designed for small team now and Staging can be taken as kind of Dev cloud environment. Eventually, we will do sub-namespace and allow full permission on sub-namespace only.